objects/Cargo.toml

```diff
@@ -19,6 +19,6 @@ default = ["std"]
 std = ["assembly/std", "crypto/std"]
 
 [dependencies]
-assembly = {  package = "miden-assembly", git = "https://github.com/0xPolygonMiden/miden-vm", branch = "next", default-features = false }
-crypto = {  package = "miden-crypto", git = "https://github.com/0xPolygonMiden/crypto", branch = "next", default-features = false }
-miden-core = {  package = "miden-core", git = "https://github.com/0xPolygonMiden/miden-vm", branch = "next", default-features = false }
+assembly = {  package = "miden-assembly", git = "https://github.com/0xPolygonMiden/miden-vm", default-features = false, rev = "914059c4e72965acbae78ca285b775ae30408d4b"  }
```

Comment:
This can be updated once https://github.com/0xPolygonMiden/miden-vm/pull/687 is merged (or we find an alternative to that)